testing-common/integration-tests/src/main/java/context/ContextTestInstrumentationModule.java

```diff
@@ -87,10 +93,14 @@ public static void methodExit(@Advice.Return(readOnly = false) boolean isInstrum
     @Advice.OnMethodExit
     public static void methodExit(
         @Advice.This KeyClass thiz, @Advice.Return(readOnly = false) int contextCount) {
-      ContextStore<KeyClass, Context> contextStore =
-          InstrumentationContext.get(KeyClass.class, Context.class);
-      Context context = contextStore.putIfAbsent(thiz, new Context());
-      contextCount = ++context.count;
+      try {
+        ContextStore<KeyClass, Context> contextStore =
+            InstrumentationContext.get(KeyClass.class, Context.class);
+        Context context = contextStore.putIfAbsent(thiz, new Context());
+        contextCount = ++context.count;
+      } catch (Throwable t) {
+        t.printStackTrace();
```

Comment:
Is this needed even though we don't have a suppress? Well IIRC default of bytebuddy is to suppress, should we disable suppression instead and let it throw?

Reply:
oops, this was just debugging code that I forgot to remove 👍